### PR TITLE
Add Obsidian to Integrations page, add some more copy on Pocket integration

### PR DESCRIPTION
**Pull request by jacksonh**

### CODE REVIEW

File: packages/web/pages/settings/integrations.tsx [hunk 1]
```diff
@@ -146,18 +146,32 @@ export default function Integrations(): JSX.Element {
...
-        icon: '/static/icons/readwise.svg',
-        title: 'Readwise',
+        icon: '/static/icons/obsidian.png',
+        title: 'Obsidian',
         subText:
-          'Readwise makes it easy to revisit and learn from your ebook & article highlights. Use our Readwise integration to sync your highlights from Omnivore to Readwise.',
+          'Obsidian is a powerful and extensible knowledge base that works on top of your local folder of plain text files. Use the Omnivore Obsidian plugin to sync articles, highlights, and notes to Obsidian.',
         button: {
-          text: readwiseConnected ? 'Remove' : 'Connect to Readwise',
+          text: `Install Logseq Plugin`,
```
Comment (sywhb) [inline packages/web/pages/settings/integrations.tsx:154]: A typo here. Should be `Install Obsidian Plugin`. Typical C&P 😄

File: packages/web/pages/settings/integrations.tsx [hunk 2]
```diff
@@ -146,18 +146,32 @@ export default function Integrations(): JSX.Element {
...
+          icon: <DownloadSimple size={16} weight={'bold'} />,
+          style: 'ctaDarkYellow',
+          action: () => {
+            router.push(`https://github.com/omnivore-app/logseq-omnivore`)
```
Comment (sywhb) [inline packages/web/pages/settings/integrations.tsx:158]: Same typo. Should be `https://github.com/omnivore-app/obsidian-omnivore`

File: packages/web/pages/settings/integrations.tsx (hunk 1 above)
Comment (jacksonh) [reply packages/web/pages/settings/integrations.tsx]: eeeekkkkkk!

**Review by sywhb — APPROVED** [review]
LGTM

**Pull request merged**